fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/IndexingMutuallyByRecords.java

```diff
@@ -371,7 +373,9 @@ private CompletableFuture<Boolean> buildNextRangeOnly(List<Range> missingRanges,
             }
             if (rangeToBuild != null) {
                 infiniteLoopProtection(rangeToBuild, missingRanges);
-                return iterateAllRanges(null,
+                final List<Object> additionalLogMessageKeyValues = new ArrayList<>(Arrays.asList(LogMessageKeys.CALLING_METHOD, "mutualMultiTargetIndex"));
+                additionalLogMessageKeyValues.addAll(fragmentLogMessageKeyValues());
+                return iterateAllRanges(additionalLogMessageKeyValues,
```

Comment:
It would probably be good to add `rangeToBuild` and `fragmentRange` like in the `range to build` log above.